zulip_botserver/zulip_botserver/server.py

```diff
@@ -33,18 +33,29 @@ def load_lib_modules():
     # type: () -> None
     for bot in available_bots:
         try:
-            module_name = 'zulip_bots.{bot}.{bot}'.format(bot=bot)
-            bots_lib_module[bot] = import_module(module_name)
+            module_name = 'zulip_bots.bots.{bot}.{bot}'.format(bot=bot)
+            lib_module = import_module(module_name)
+            bots_lib_module[bot] = lib_module
         except ImportError:
             print("\n Import Error: Bot \"{}\" doesn't exists. Please make sure you have set up the flaskbotrc "
                   "file correctly.\n".format(bot))
-            sys.exit(1)
+
+def get_bot_lib_module(bot):
+    # type: (str) -> Any
+    if bot in bots_lib_module.keys():
+        return bots_lib_module[bot]
+    return None
 
 app = Flask(__name__)
 
 @app.route('/bots/<bot>', methods=['POST'])
 def handle_bot(bot):
     # type: (str) -> Union[str, BadRequest]
+    lib_module = get_bot_lib_module(bot)
+    if lib_module is None:
+        return BadRequest("Either the configuration for bot {} is missing or the Bot Handler Class cannot be "
+                          "fetched.".format(bot))
+
```

Comment:
On another note, I think the error message passed to `BadRequest` is misleading. Not being able to import the bots could mean a lot of things (the `zulip_bots` package isn't installed, they misspelled the name of the bot on the command-line, etc), but so far, this error doesn't have anything to do with finding the handler class. It has more to do with the fact that it can't find the source code for the bot to begin with. So we should mention that in the error message plus recommend making sure that the `zulip_bots` package is installed. Maybe, like so:

```
BadRequest("Can't find the configuration for bot {}. Make sure that the `zulip_bots` package is installed!".format(bot)))
```
or something! I mean if we can't find the whole module to begin with, there's no point in mentioning a handler class. So, I have a feeling it could confuse people not familiar with how the bots are written. Let me know what you think! :)